bowtie/tests/miniatures.py

```diff
@@ -137,26 +137,48 @@ def links(dialect: Dialect):
     return lambda schema, registry: lambda instance: None
 
 
-@fake(name="versioned", version="1.0")
-def version_1(dialect: Dialect):
+def foo_v1(dialect: Dialect):
```

Comment:
Just inline these 2 (foo_v1 and foo_v2) I'm not sure having them be separate functions is benefiting much. Otherwise seems good to merge.

Reply:
Name it something reasonable please too, `foo` is always a terrible name for a general thing. Probably `has_buggy_v1` or something similar.